model/authorizations/v1/resource_review_request_type.model

```diff
@@ -28,4 +28,8 @@ struct ResourceReviewRequest {
 	// If true, in the case when all subscriptions in organization are permitted, response will *not* include
 	// these subscriptions' ID, but organization only.
 	ReduceClusterList Boolean
+
+	// A comma-separated list of subscription statuses.
+	// Subscriptions with these statuses will be excluded from results.
+	excludeSubscriptionStatuses string
```

Comment:
I'd say it would be better to use a list of enum values (or strings):

```
enum SubscriptionStatus {
  // Explain what 'foo' is.
  Foo

  // Explan what 'bar' is.
  Bar
}

struct ResourceReviewRequest {
  // Subscriptions with these statuses will be excluded from results.
  ExcludeSubscriptionStatuses []SubscriptionStatus
}
```

Reply:
Does it make parameter a required one?

Reply:
No. There is currently no mechanism in the model/metamodel/SDK to mark a parameter as required.